crates/proto/src/dnssec/dnssec_dns_handle/mod.rs

```diff
@@ -917,109 +1020,6 @@ fn verify_rrsig_with_keys(
     }
 }
 
-/// TODO: I've noticed upstream resolvers don't always return NSEC responses
-///   this causes bottom up evaluation to fail
```

Comment:
I notice this TODO is being dropped and just wanted to confirm explicitly it's no longer applicable. If so, might be nice to mention in the commit message.